value-streams.md

```diff
@@ -7,205 +8,93 @@ A CloudBees DevOptics Value Stream models a complex continuous delivery process
 * Find components ready for testing
 * View contributing components
 
-## Value Stream Mapping for Microservices
-Using the two applications used in the [CloudBees Core Pipeline Workshop](https://github.com/cloudbees-days/cloudbees-core-workshop), we will show how you can use DevOptics Value Streams to map the software delivery process of loosely coupled microservices as separated streams as part of a holistic value stream.
+In the first set exercises we will setup a few sample Jenkins pipelines by [creating a GitHub Org Folder Pipeline project](#create-a-github-organization-folder-pipeline-project), get an overview of the [basic fundamentals of CloudBees DevOptics](#login-to-cloudbees-devoptics), and learn how to [configure a Value Stream in DevOptics mapped to a sample Jenkins pipeline](#value-stream-visual-editor). But first we have to [add some credentials to our Team Master](#add-github-credentials-to-your-team-master) - a **Username with password** credential for the GitHub Org Folder Pipeline project.
+
+## Add GitHub Credentials to Your Team Master
+
+We must switch to the Jenkins classic UI to complete the steps in this exercise.
+
+1. Click the ***Go to classic*** button at the top of common section of Blue Ocean’s navigation bar. <p><img src="img/intro/go_to_classic.png" width=800/>
```

Comment:
Click the ***Go to classic*** button at the top of common section - reads a bit weird

Perhaps:
Click the ***Go to classic*** button at the top of the common section